onnxruntime/core/providers/xnnpack/xnnpack_execution_provider.cc

```diff
@@ -124,12 +126,19 @@ void XnnpackExecutionProvider::RegisterAllocator(AllocatorManager& allocator_man
   }
 }
 
+// ops are not lay-out sensitive and does not defined in
+// onnx-domain will be created dynamicly
+static bool RequestDynamicSchema(absl::string_view op_type) {
+  static const InlinedHashSet<absl::string_view> dynamic_schema_set = {"QLinearSoftmax"};
+  return dynamic_schema_set.contains(op_type);
+}
+
```

Comment:
This may need to be refined as it should also support the target node of a QDQ node group being mapped to the xnnpack internal operator. In that case you'd have a Softmax node in a Type::QDQGroup NodeUnit, and that could be fused into a xnnpack QLinearSoftmax node.

Reply:
In the second call of `GetCapability`, a node-group or any customized fusion would be obscured.  we only need to put the finalized opname here for dynamic schema. In that case, Xnnpack can't see QDQ node groups. Does it make sense?

Reply:
Not quite. Don't we convert QDQ node groups to a quantized op (if possible) in the second call to GetCapability as the first call must leave as ONNX ops so layout conversion works? So that second call will have QDQ node groups that we need to decide if we can handle by fusing to a node with a quantized kernel. 

Naively I expected we'd do a lookup using the NodeUnit (so both QDQ node groups and individual nodes are handled) to see if the EP has an internal kernel that can handle it. The schema is generated for the fused node, and the fused node in this example would be the fusion of DQ -> Softmax -> Q nodes.

Reply:
The key generating rules will be node_unit.OpType() or "QLinear"+node_unit.OpType() for QDQGroup.

Reply:
> The key generating rules will be node_unit.OpType() or "QLinear"+node_unit.OpType() for QDQGroup.

You can not use anything from `absl` namespace, otherwise our no Abseil build would not work. Use `std::string_view` in this case.

Reply:
Okay